app/src/main/java/com/mk/tasky/agenda/presentation/photoviewer/PhotoViewerScreen.kt

```diff
@@ -0,0 +1,84 @@
+package com.mk.tasky.agenda.presentation.photoviewer
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material.icons.filled.DeleteOutline
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.mk.tasky.R
+import com.mk.tasky.ui.theme.Black
+import com.mk.tasky.ui.theme.White
+
+@Composable
+fun PhotoViewerScreen(
+    onBack: (Boolean) -> Unit,
```

Comment:
Nitpick, but what does the Boolean represent? If it doesn't get clear from the lambda name, I like to explicitly name lambda parameters: `onBack: (clearName: Boolean) -> Unit`

Reply:
Fixed!